docs/serve.json

```diff
@@ -0,0 +1,9 @@
+{
+  "public": "build",
+  "rewrites": [
+    {
+      "source": "/docs/rasa/next/:rest",
+      "destination": "/:rest"
+    }
+  ]
```

Comment:
after testing locally, I had to add the following config:
```
"redirects": [
    { "source": "/docs/rasa/next/js/:rest", "destination": "/js/:rest" },
    { "source": "/docs/rasa/next/img/:rest", "destination": "/img/:rest" },
    { "source": "/docs/rasa/next/spec/:rest", "destination": "/spec/:rest" }
  ]
```
to make it work for the static files

Reply:
Hmm it worked for me. OK I think I'll try adding `netlify-cli` instead of `serve` here, since it handles wildcard redirects. If I place a `_redirects` file under `/docs/static` with this syntax, it redirects sub-paths as well, and it will be interesting to see if it picks up in the deploy as well
```sh
/docs/rasa/next/* /:splat 200!
```
```sh
# example logs...
◈ Rewrote URL to /8c830e4d.75ee2463.js
◈ Rewrote URL to /10ac8ce8.459682b2.js
◈ Rewrote URL to /735d0569.987e3f1d.js
◈ Rewrote URL to /90534a25.6cb67509.js
◈ Rewrote URL to /img/dialogflow_export.png
◈ Rewrote URL to /img/dialogflow_export_2.png
◈ Rewrote URL to /3.aa639465.js
```